src/main/kotlin/day5/CollectionMain.kt

```
@@ -0,0 +1,46 @@
+
+//    val wizard = Wizard()
+//
+//    wizard.attack()
+//    wizard.fireball { damage: Int ->
+//        println(damage)
+//    }
+}
+
+fun damage(num: Int): Unit {
+
+}
```

detekt (1.23.7)

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)